apps/mailcatcher/templates/services/app/deploy.yml.j2

```diff
@@ -9,12 +9,16 @@ metadata:
   namespace: "{{ namespace_name }}"
 spec:
   replicas: 1  # number of pods we want
+  selector:
+    matchLabels:
+      app: mailcatcher
+      service: app
+      deploymentconfig: "mailcatcher-app"
```

Comment:
We still use this label ? I think you should rename it in `deployment` here and line 22

Reply:
Good catch, thanks!
Yes, this label is used by the service.